doc/_dev-guide/chapter_4/section_2/README.md

```diff
@@ -136,22 +262,58 @@ brew install \
    ```
 
-3. Extend the search path of pkg-config to use the homebrew version of openssl, net-snmp and pkgconfig
+   Extend the search path of pkg-config to use the homebrew version of openssl, net-snmp and pkgconfig
 
    ```shell
    export PKG_CONFIG_PATH=${HOMEBREW_PREFIX}/opt/openssl/lib/pkgconfig:${HOMEBREW_PREFIX}/opt/net-snmp/lib/pkgconfig:${HOMEBREW_PREFIX}/lib/pkgconfig:${PKG_CONFIG_PATH}
    ```
 
-4. Some of the modules will not, or will incorrectly provide pkg-config support (even if added to PKG\_CONFIG\_PATH, e.g. libmaxminddb), to help the automatic configurations find them you can add brew to `CFLAGS`, `CPPFLAGS`, and `LDFLAGS`
+   Some of the modules will not, or will incorrectly provide pkg-config support (even if added to `PKG\_CONFIG\_PATH`, e.g. libmaxminddb), to help the automatic configurations find them you can add brew to `CFLAGS`, `CPPFLAGS`, and `LDFLAGS`
 
    ```shell
    export CFLAGS="-I${HOMEBREW_PREFIX}/include/ ${CFLAGS}"
    export CPPFLAGS="${CFLAGS} ${CPPFLAGS}"
    export LDFLAGS="-L${HOMEBREW_PREFIX}/lib ${LDFLAGS}"
    ```
 
-   **Note:** It could also happen that you must provide here further library inlcude and lib paths, e.g. for openssl 1.1.x, etc.
-   {: .notice}
+#### Using MacPorts
+
+1. MacPorts is using a fixed location for storing its data, currently it is the `/opt/local` folder.
+
+   In a nutshell, you can setup the env manually like this
+
+   ```shell
+   export MACPORTS_PREFIX=/opt/local
+   export MACPORTS_REPOSITORY=${MACPORTS_PREFIX}
+   export MANPATH=${MACPORTS_PREFIX}/share/man:${MANPATH}
+   export INFOPATH=${MACPORTS_PREFIX}/share/info:${INFOPATH}
+   export PATH=${MACPORTS_PREFIX}/bin:${MACPORTS_PREFIX}/sbin:${MACPORTS_PREFIX}/opt/python/libexec/bin:${PATH}
+   ```
+
+2. Force the building process to use bison, net-snmp (and libnet if) installed through MacPorts instead of provided by Apple Developer Tools and macOS.
+
+   Add them to the `${PATH}`
+
+   ```shell
+   export PATH=${MACPORTS_PREFIX}/opt/bison/bin:${MACPORTS_PREFIX}/opt/libnet/bin:${MACPORTS_PREFIX}/opt/net-snmp/bin:${PATH}
+   ```
+
+   Extend the search path of pkg-config to use the MacPorts version of openssl, net-snmp and pkgconfig
+
+   ```shell
+   export PKG_CONFIG_PATH=${MACPORTS_PREFIX}/opt/openssl/lib/pkgconfig:${MACPORTS_PREFIX}/opt/net-snmp/lib/pkgconfig:${MACPORTS_PREFIX}/lib/pkgconfig:${PKG_CONFIG_PATH}
+   ```
+
+   Some of the modules will not, or will incorrectly provide pkg-config support (even if added to `PKG\_CONFIG\_PATH`, e.g. libmaxminddb), to help the automatic configurations find them you can add brew to `CFLAGS`, `CPPFLAGS`, and `LDFLAGS`
+
+   ```shell
+   export CFLAGS="-I${MACPORTS_PREFIX}/include/ ${CFLAGS}"
+   export CPPFLAGS="${CFLAGS} ${CPPFLAGS}"
+   export LDFLAGS="-L${MACPORTS_PREFIX}/lib ${LDFLAGS}"
+   ```
+
+**Note:** It could also happen that you must provide here further library inlcude and lib paths, e.g. for openssl 1.1.x, etc.
```

Comment:
[optional] "Providing further library paths might be necessary" is a more natural wording